packages/grafana-data/src/field/displayProcessor.ts

```diff
@@ -125,29 +122,6 @@ function toStringProcessor(value: any): DisplayValue {
   return { text: _.toString(value), numeric: toNumber(value) };
 }
 
-export function getColorFromThreshold(value: number, thresholds: Threshold[], theme?: GrafanaTheme): string {
```

Comment:
FYI -- this code was only used in tests... but not used in any real code.  I suspect it was left over.  To get the tests to work we need have the thresholds sorted and -Infinity set